data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTypes.java

```diff
@@ -68,4 +70,8 @@ public class BeaconRestApiTypes {
   public static final ParameterMetadata<String> ID_PARAMETER =
       new ParameterMetadata<>(
           PARAM_ID, CoreTypes.STRING_TYPE.withDescription(PARAM_VALIDATOR_DESCRIPTION));
+
+  public static final ParameterMetadata<String> STATUS_PARAMETER =
+      new ParameterMetadata<>(
+          STATUS, CoreTypes.STRING_TYPE.withDescription(PARAM_STATUS_DESCRIPTION));
```

Comment:
i think we've got an enum and could make this a parameter of that enumeration type...